src/start_server.ts

```diff
@@ -347,11 +354,19 @@ export function getApp(options: ServerOptions): express.Express {
   }
 
   if (options.compile === 'auto' || options.compile === 'always') {
-    app.use('*', babelCompile(options.compile === 'always'));
+    // Compile all files, except index.html
+    app.use(/\/.+/, babelCompile(options.compile === 'always'));
```

Comment:
This was required to not break serving `index.html`. When using 2 transformers, calling `res.end()` twice will render an invalid response and the browser shows a white page.